nginx.json

```diff
@@ -0,0 +1,44 @@
+{
+	"nginx": {
+		"description": "Open source reverse proxy server for HTTP, HTTPS, SMTP, POP3, and IMAP protocols, as well as a load balancer, HTTP cache, and a web server.",
```

Comment:
```suggestion
		"description": "Open source web server for HTTP, HTTPS, SMTP, POP3, and IMAP protocols, as well as a load balancer, HTTP cache, and a reverse proxy server.<p>Advanced users only: You must <a href='https://gist.github.com/JasonTarka/63d632b5633fb0840cdfb3b56a14d388'>add a default config file to your "Config Storage" share.</a>.</p><p>Based on the official Nginx docker image: <a href='https://hub.docker.com/_/nginx' target='_blank'>https://hub.docker.com/_/nginx</a>.</p>",
```

Reply:
@JasonTarka I know it's been some time but we now have a different format we are attempting to add to all existing and future Rock-ons. Linking to the docker image used is part of that format.